docs/migration-guides/external-providers-primitives-v2.mdx

```diff
@@ -0,0 +1,216 @@
+---
+title: Migrate provider interfaces from backend.run to primitives
+description: Migrate to the primitives interface in external providers
+
+---
+
+# Migrate provider interfaces from `backend.run` to primitives
+
+## Motivation and context
+
+The evolution of quantum algorithms and scaling of quantum processing units (QPUs) motivated the
+introduction of the Qiskit [primitives interface](/guides/primitives) to provide a higher-level abstraction for
+accessing QPUs. This interface is optimized for two core tasks in quantum algorithm development: expectation value estimation (Estimator) and
+circuit sampling (Sampler).
+
+After a time of co-existence of the `backend.run` model and primitives model, when the primitives
+evolved to the updated V2 primitives, the `backend.run` model was finally deprecated in favor of
+these newer interfaces to streamline quantum workload execution.
+
+The existing collection of migration guides helps users [transition to the Qiskit Runtime provider](/migration-guides/qiskit-runtime) and
+[update their user code to the V2 primitives interface](/migration-guides/v2-primitives) in time for the upcoming removal of `backend.run`.
+To complement that, this guide supports external providers (both existing and upcoming)
+in implementing their own version of the primitives V2 interface (Estimator or Sampler) that integrates seamlessly into the
+standard Qiskit workflow.
+
+These implementations can be used to wrap any custom provider hardware access function
+(for example: `execute_workload(QPU)` or `resource.access()`) or local simulator,
+as long as the final inputs and outputs conform to the established standards set by the primitive interfaces.
+
+## If your provider already implemented `backend.run`
```

Comment:
I'm a bit confused by this section... this reads like the target audience is people who _use_ 3rd party providers, and not the providers themselves (e.g. users of `IonQProvider` and not IonQ itself).

Reply:
I am not too set on having this section, to be honest, that's why I mentioned in the PR description that I wanted a second opinion on this approach. I based it on the only kind of external (still mostly IBMers) provider that I know implemented primitives, that is: https://github.com/qiskit-community/qiskit-aqt-provider/tree/master. They based their primitives on the backend primitives as shown in this example, and I thought it might be a good shortcut for other providers too, but I also think it might be too hacky. If you don't think it adds any value, it could be removed.

Reply:
I think we can point out backend primitives as an example, but don't explicitly say they should use it. It's also to their benefit to optimize for their hardware instead of using an out-of-box implementation.

This section would actually be quite helpful in the existing `backend.run` migration guide. If, for example, someone is using `backend.run` across Qiskit Runtime + Provider Foo and want to migrate to primitives. They can use this  as an example to write to primitives interface even if Foo hasn't migrated yet.